src/EA.Weee.Web.Tests.Unit/Areas/Producer/Controllers/ProducerControllerTests.cs

```diff
@@ -744,6 +754,43 @@ public void DownloadSubmission_Get_ShouldHaveSmallProducerSubmissionContextAttri
             methodInfo.Should().BeDecoratedWith<HttpGetAttribute>();
         }
 
+        [Fact]
+        public void ProducerController_ShouldHaveAuthorizeRouteClaimsAttribute()
+        {
+            // Arrange
+            var typeInfo = typeof(ProducerController);
+
+            // Act
+            var attribute = (AuthorizeRouteClaimsAttribute)Attribute.GetCustomAttribute(typeInfo, typeof(AuthorizeRouteClaimsAttribute));
```

Comment:
@will-hope98  similar test for the ProducerSubmissionController I think

Reply:
done